app/controllers/posts.rb

```diff
@@ -0,0 +1,64 @@
+get '/' do
+  redirect '/index'
+end
+
+get '/index' do
+  @questions = Question.all
+  @answer_counts = @questions.map{ |question| question.answers.count }
+  @scores = @questions.map{ |question| question.score }
```

Comment:
the calculation for both `@scores` and `@answer_counts` should probably live on the `Question` model since it's performing a calculation on our data. when you have a chance to refactor (if you do), definitely extract this out!  skinny controllers - fat models (is more preferable than fat controllers)